src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.ui;
+
+/**
+ * Formatting text for display
+ */
+
+public class Formatter {
+	
+	/** A decorative prefix added to the beginning of lines printed by AddressBook */
+	private static final String LINE_PREFIX = "|| ";
+	
```

Comment:
@nightism Tabs :(

FWIW I use the "show whitespace" options in eclipse to check for these stuff since the misconfigured autoindent can sometimes add tabs (http://stackoverflow.com/a/27828467)